doc/developers/advanced_installation.rst

```diff
@@ -242,12 +242,20 @@ The above commands assume that you have the Python installation folder in your
 PATH environment variable.
 
 You will need `Build Tools for Visual Studio 2017
-<https://visualstudio.microsoft.com/de/downloads/>`_.
+<https://visualstudio.microsoft.com/downloads/>`_.
+
+.. warning::
```

Comment:
You've done something strange in trying to merge in changes from master. Please try to merge in the latest master again